api/nodes/views.py

```diff
@@ -2382,6 +2382,8 @@ class NodeProvidersList(JSONAPIBaseView, generics.ListAPIView, NodeMixin):
     view_category = 'nodes'
     view_name = 'node-providers'
 
+    ordering = ('-pk',)
```

Comment:
This view is for [NodeProviders](https://github.com/CenterForOpenScience/osf.io/blob/develop/api/nodes/views.py#L2246), which don't have an `id` field!